### DISCUSSION TITLE
Is there a way to filter a field relatively to the value of another?

### DISCUSSION THREAD

**Discussion by obezzad** [directus/directus#14036] [Archive]
1. Is it possible to filter a field by comparing its value to another field?
![image](https://user-images.githubusercontent.com/2561446/175003191-b6cdd5fc-5446-41cd-b093-8be3528b1827.png)
2. What about comparing **Cost** relatively to an aggregate function (sum, maximum, average...)? (e.g. the **Cost** is max)
3. Is there a workaround to show another field (e.g. **Type**) in Insights tab while filtering with the **cost** being max/min...?

**Comment by rijkvanzanten** [comment]
Not currently. Give an upvote to https://github.com/directus/directus/discussions/2993 🙂
